Lib/test/test_dataclasses.py

```diff
@@ -858,6 +859,7 @@ def bar(self) -> int:
         self.assertEqual(C(10).bar, 5)
         self.assertEqual(C(10).i, 10)
 
+    @requires_docstrings
     def test_post_init(self):
```

Comment:
I am not sure that this test tests *only* docstrings.

Reply:
I don't see how it tests doc strings at all. If there's some failure related to lack of docstrings, this will need to be refactored so that the non-docstring-related parts (if any) are still executed.